sql/item_latents.sql

```diff
@@ -2791,3 +2792,11 @@ INSERT INTO `item_latents` VALUES (28235,76,25,26,2);   -- Hachiya Kyahan: Dusk
 INSERT INTO `item_latents` VALUES (28256,76,25,26,2);   -- Hachiya Kyahan +1: Dusk to dawn: MOVE_SPEED_GEAR_BONUS+25%
 INSERT INTO `item_latents` VALUES (28445,23,10,14,0);   -- Shetal Stone ATT +10 No Food Active
 INSERT INTO `item_latents` VALUES (28445,68,10,14,0);   -- Shetal Stone EVA +10 No Food Active
+
+-- Nesanica ring
+INSERT INTO `item_latents` VALUES (28567,368,10,13,2);    -- While Sleeping: REGAIN +10
+INSERT INTO `item_latents` VALUES (28567,368,10,13,19);   -- While Sleeping: REGAIN +10
+INSERT INTO `item_latents` VALUES (28567,368,10,13,193);  -- While Sleeping: REGAIN +10
+INSERT INTO `item_latents` VALUES (28567,370,1,13,2);     -- While Sleeping: REGEN +1
+INSERT INTO `item_latents` VALUES (28567,370,1,13,19);    -- While Sleeping: REGEN +1
+INSERT INTO `item_latents` VALUES (28567,370,1,13,193);   -- While Sleeping: REGEN +1
```

Comment:
I'm away and can't look up what the last param here is going, what is changing here with 2, 19 and 193? On this single item, etc. (Same pattern with Opo-opo Necklace)

Reply:
These correlate to Sleep(2), Sleep_II(19), and Lullaby(193) status effect IDs respectively. LatentID "13" is for if a player has a certain status effect.

Reply:
> These correlate to Sleep(ID:2), Sleep_II(ID:19), and Lullaby(ID:193) status effect IDs respectively. LatentID "13" is for if a player has a certain status effect.

exactly as you said